src/introduction/index.md

```diff
@@ -3,20 +3,19 @@
 
 Welcome to the Rust FFI Guide, i.e. **using unsafe for fun and profit**.
 
-> **Note:** I'm going to assume you're already familiar with the [Rust][rust]
-> language and have a relatively recent version of the compiler installed. If
-> you're a little rusty, you might want to skim through [The Book][book] to
-refresh your memory.
+> **Note:** This guide assumes familiarity with the [Rust][rust] language and
+> that a recent version of the compiler is installed. If you're a little rusty,
+you might want to skim through [The Book][book] to refresh your memory.
 >
-> You'll also need to know some basic C/C++ or Python, as I'll be largely using
-> that in my examples.
```

Comment:
I like this entire block. It makes the guide sound more professional while still explaining the skills required.